packages/amplication-client/src/Entity/EntityForm.tsx

```diff
@@ -81,6 +82,14 @@ const EntityForm = React.memo(({ entity, resourceId, onSubmit }: Props) => {
     return sanitizedDefaultValues as EntityInput;
   }, [entity]);
 
+  const { data: resourceSettings } = useQuery<{
```

Comment:
I suggest to move this query into one of our hooks and use it from there, instead of implementing each time

Reply:
what about that ?

Reply:
Done